scripts/linters/run_lint_checks.py

```diff
@@ -643,7 +643,11 @@ def main(args: Optional[List[str]] = None) -> None:
         return
 
     read_files(all_filepaths, namespace=namespace)
-    files: Dict[str, List[str]] = multiprocessing.Manager().dict()
+    # Here we use MyPy ignore because multiprocessing.Manager().dict()
```

Comment:
Can we make it compatible? Do we need to cast it? What's the type of multiprocessing.Manager().dict()?